.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+echo '--- :evergreen_tree: Configuring Environment'
+set -euo pipefail
+
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successful"
+
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
```

Comment:
The purpose is to have a consistency in the names used to tag the containerized images and pushed correctly to the repo, then allows to everyone to easily pull those images.

Reply:
It is a little more than consistency. The `SANITIZED_TAG` is necessary because the list of characters allowed in `git` references is more permissive than the list of characters allowed in docker image names.